src/core/constraint.jl

```diff
@@ -14,99 +14,140 @@ end
 
 # Generic thermal limit constraint
 "`p[f_idx]^2 + q[f_idx]^2 <= rate_a^2`"
-function constraint_thermal_limit_from(pm::AbstractPowerModel, n::Int, f_idx, rate_a)
+function constraint_thermal_limit_from(pm::AbstractPowerModel, n::Int, f_idx, rate_a; constraint_name=nothing)

```

Comment:
We can use the string as the default value

```suggestion
function constraint_thermal_limit_from(pm::AbstractPowerModel, n::Int, f_idx, rate_a; constraint_name = "thermal_limit_from[$f_idx]")
```